com.arc.embeddedcdt/src/com/arc/embeddedcdt/launch/Launch.java

```diff
@@ -436,24 +419,25 @@ public void launch(ILaunchConfiguration configuration, String mode, final ILaunc
 
                         String gdb_init = "";
 
-                        if (isAshling(configuration)) {
+                        String defaultGDBHost = configuration.getAttribute(
+                                LaunchConfigurationConstants.DEFAULT_GDB_HOST, "");
```

Comment:
There is definetely a bug here. DEFAULT_GDB_HOST is "localhost" and should be used directly not as a name of attribute. In fact I don't see corresponding `setAttribute` invocation at all. That seems to be a bug in a master branch already, can you make additional commit that fixes it.